index.js

```diff
@@ -17,11 +18,23 @@ const action = async context => {
 	const split = context.config.get('path').split('/');
 	const bucket = split.shift();
 	const filename = path.basename(filePath);
+	const extension = path.extname(filename);
+	const objectKey = path.join(split.join('/'), filename);
+
+	const contentTypes = new Map([
```

Comment:
Just put it outside this function above line 7.

Reply:
Done here: 7f86ed943851